client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -69,26 +69,26 @@ define({
 "No matching datasets found" :
     "Aucunes données correspondantes trouvées",
```

Comment:
Not sure I remember when we have this kind of message... "Aucun jeux de données appropriés trouvés"